dist/ci/travis_script.sh

```diff
@@ -33,6 +33,7 @@ if test -z "$SUBTEST"; then
       bundle exec rails test:spider
       ;;
     linter)
+      bundle exec rake db:structure:verify
```

Comment:
should this be together with the style linters? I mean that rubocop, haml_lint and jshint are checking code conventions, which I would say is not related with what the new linter makes. I would put it in a different test suite. :thinking: